### ISSUE TITLE
Create Marathon 1.3.7

### ISSUE THREAD

**Issue by timcharper** [mesosphere/marathon#4925]
Already picked:

- a7f4138a1609e410901095cce6cdcc81541206d8 Escape leader-latch lock (addresses deadlock) Addresses part of #4624

**Comment by janisz** [comment]
When 1.3.7 will be released?

**Comment by jasongilanfarr** [comment]
I believe we need #4968 in 1.3.

**Comment by janisz** [comment]
@unterstein Release page is not complete it shows only 4 solved issues while there were more:
```bash
git log --date=short --graph --pretty=format:'%h %d %s (%an)' --abbrev-commit v1.3.6..v1.3.7 
```
* c2d6089  (tag: v1.3.7) Setting version to 1.3.7 (Johannes Unterstein)
* fb4aeb9  Quote some vars and make sed more explicit (#4890) (Lukas Lösche)
* dafd0df  workaround for zip64 incompat with shebang-prefixed jars (James DeFelice)
* e888d16  Fixes #4637 Allow filtering SSE events by types (#4936) (janisz)
* eaaa994  Parse JSON event only once before broadcast (#4927) (janisz)
* 7b346de  Split long lines (#4926) (janisz)
* a7f4138  escape leader-latch lock (targets 1.3) (Brad Peters)
* a05a01b  Releases/1.3 async task tracker (#4912) (janisz)
* 90843e4  Include TASK_FINISHED as failure state when upgrading (#4865) (fengyehong)
* d3351ea  Cherry-picked Handle spaces in arguments correctly (#4887) (janisz)
* f72df1b  Cherry-picked  MigrationTo1_1 class to handle broken app groups (#4711) (#4772) (Aleksey Dukhovniy)
* 4f75209  Allow Zookeeper Connection Timeout to be configured. (#4685) (Jason Gilanfarr)
* 850db7f  Allow Zookeeper Connection Timeout to be configured. (Jason Gilanfarr)
* 116992d  Update #1428 to use MARATHON_CMD to avoid breaking other integration (Jason Gilanfarr)
* 689b250  Update changelog wording for 1.3.5 (Jason Gilanfarr)
* 51f103e  Fixes #1428 Converts command arguments to environment variables and back again (merge to release/1.3) (#4644) (brad-peters)

**Comment by unterstein** [comment]
Thanks @janisz, I updated the release log :)

DC/OS PRs are open as well and github pages documentation is updated.
